### PR TITLE
Finished login 

### PR DESCRIPTION
**Pull request by repovic48**

### CODE REVIEW

File: WebApplication/Startup.cs [hunk 1]
```diff
@@ -28,19 +32,37 @@ public Startup(IConfiguration configuration, IWebHostEnvironment environment)
...
+               });
             services.AddDbContext<HealthCareSystemDbContext>(options =>
             {
                 var connectionString = CreateConnectionStringFromEnvironment();
                 Console.WriteLine(connectionString);
 
                 options.UseNpgsql(
-                    connectionString,
-                    x => x.MigrationsAssembly("Backend").EnableRetryOnFailure(5,
+                    CreateConnectionStringFromEnvironment(),
```
Comment (milicasiriski) [inline WebApplication/Startup.cs]: use connectionString variable (this was changed when you resolved conflicts)

File: WebApplication/Startup.cs [hunk 2]
```diff
@@ -52,6 +74,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IReportRepository, ReportDatabaseSql>();
             services.AddScoped<ISpecializationRepository, SpecializationDatabaseSql>();
             services.AddScoped<ISurveyRepository, SurveyDatabaseSql>();
+            services.AddScoped<Backend.Repositorys.Interfaces.IAdminRepository, AdminDatabaseSql>();
```
Comment (milicasiriski) [inline WebApplication/Startup.cs]: Just import IAdminRepository from backend, you probably already have so just delete the namespace path part

File: WebApplication/Startup.cs [hunk 3]
```diff
@@ -114,8 +147,12 @@ private string CreateConnectionStringFromEnvironment()
             var user = userInfo[0];
             var password = userInfo[1];
             var database = databaseUri.LocalPath.TrimStart('/');
+            // Change this if user and password are different
+            string user = Environment.GetEnvironmentVariable("DATABASE_USERNAME") ?? "postgres";
+            string password = Environment.GetEnvironmentVariable("DATABASE_PASSWORD") ?? "root";
```
Comment (milicasiriski) [inline WebApplication/Startup.cs]: Delete this, also a mistake from conflict resolution and this is causing your build errors

**Review by milicasiriski — CHANGES_REQUESTED** [review]
Fix the Start-up and build errors first to make sure everything is ok, then we'll look at the rest.

File: project-generated-code-backend/Backend/Model/Accounts/Admin.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+        [JsonConstructor]
+        public Admin(string serialNumber, string name, string surname, string id, DateTime dateOfBirth, string contact, string email,
+            Address address, string password)
+            : base(serialNumber, name, surname, id, dateOfBirth, contact, email, address, password)
+        {
+
+        }
+
+        [JsonConstructor]
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Model/Accounts/Admin.cs]: Delete json constructor annotation, there can be only one

File: project-generated-code-backend/Backend/Model/Accounts/Physician.cs [hunk 5]
```diff
@@ -21,12 +21,12 @@ public class Physician : Account
         public virtual TimeInterval WorkSchedule { get; set; }
         public virtual List<TimeInterval> VacationTime { get; set; }
         [NotMapped] public virtual List<Specialization> Specialization { get; set; }
-        
+        /*
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Model/Accounts/Physician.cs]: You can delete this if no one is using it

File: project-generated-code-backend/Backend/Repository/Generic/IAdminRepository.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApplication.Backend.Repositorys.Interfaces
+{
+    public interface IAdminRepository :IGenericRepository<Admin>
+    {
+        //5555555555555555555555555555555555555555
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Repository/Generic/IAdminRepository.cs]: hahah what is this?

**Review by milicasiriski — APPROVED** [review]
Good PR, a couple of comments are left here and there.

**Review by milicasiriski — APPROVED** [review]

**Review by Tanjaa1 — APPROVED** [review]

**Pull request merged**